POSTHOG_SETUP.md

```
@@ -0,0 +1,121 @@
+# PostHog Configuration
+POSTHOG_KEY="phc_your_project_key_here"
+POSTHOG_HOST="https://us.i.posthog.com"
+NEXT_PUBLIC_POSTHOG_KEY="phc_your_project_key_here"
+NEXT_PUBLIC_POSTHOG_HOST="https://us.i.posthog.com"
+```
+
+You can find your project API key in your PostHog project settings.
+
+## Files Added/Modified
+
+### Server-side PostHog (`src/server/posthog.ts`)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)